### PR TITLE
Backend actions

### PR DESCRIPTION
**Pull request by AinoraZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ BillChop is an online tool that allows you to keep track of your payments, ascer
 
 ## Github Actions
 Frontend: ![](https://github.com/TeamInterject/BillChop/workflows/Frontend-CI/badge.svg)
+Backend: ![](https://github.com/HuCaresTeam/schedulearn/workflows/Backend-CI/badge.svg)
```
Comment (tmaciulis22) [inline README.md]: HuCaresTeam?

File: billchop-be/billchop-be/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ public IEnumerable<WeatherForecast> Get()
             return Enumerable.Range(1, 5).Select(index => new WeatherForecast
             {
                 Date = DateTime.Now.AddDays(index),
-                TemperatureC = rng.Next(-20, 55),
+                TemperatureC = rng.Next(-20, 60),
```
Comment (tmaciulis22) [inline billchop-be/billchop-be/Controllers/WeatherForecastController.cs:33]: What is this controller?

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AinoraZ) [reply README.md]: My bad, old team :D

File: billchop-be/billchop-be/Controllers/WeatherForecastController.cs (hunk 2 above)
Comment (AinoraZ) [reply billchop-be/billchop-be/Controllers/WeatherForecastController.cs:33]: It's just the default that comes from the API template. Changed insignificant line of code to trigger the workflow.

**Review by tmaciulis22 — APPROVED** [review]

**Pull request merged**